actor/builtin/storagemarket/storagemarket.go

```diff
@@ -57,9 +57,9 @@ type Actor struct{}
 type State struct {
 	Miners cid.Cid `refmt:",omitempty"`
 
-	// TotalCommitedStorage is the number of sectors that are currently committed
-	// in the whole network.
-	TotalCommittedStorage *big.Int
+	// TODO: Determine correct unit of measure. Could be denominated in the
```

Comment:
Would you mind linking an existing github issue in this TODO, or making a new one and linking it if one does not already exist?

Reply:
Done!